packages/react-components/react-datepicker-compat/src/components/DatePicker/useDatePicker.tsx

```diff
@@ -165,12 +169,23 @@ export const useDatePicker_unstable = (props: DatePickerProps, ref: React.Ref<HT
           if (!date || isNaN(date.getTime())) {
```

Comment:
The comment on line `164` has a typo, it reads `is and invalid` whereas it should read `is an invalid`.